k9mail-library/src/test/java/com/fsck/k9/mail/store/imap/ImapConnectionTest.java

```diff
@@ -632,11 +715,11 @@ public void open_withNegativeResponseToStartTlsCommand_shouldThrow() throws Exce
     public void open_withCompressDeflateCapability_shouldEnableCompression() throws Exception {
         settings.setUseCompression(true);
         MockImapServer server = new MockImapServer();
-        simplePreAuthAndLoginDialog(server, "COMPRESS=DEFLATE");
+        simplePreAuthAndLoginDialog(server, "", "COMPRESS=DEFLATE");
```

Comment:
Please introduce a new method rather than adding a new argument that is the same non-obvious value almost everywhere.